great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -1628,16 +1631,64 @@ def _plot_column_domain_charts(
         plot_mode: PlotMode,
         sequential: bool,
     ) -> Tuple[List[alt.VConcatChart], List[alt.Chart]]:
+        column_based_expectation_configurations_by_type: Dict[
+            str, List[ExpectationConfiguration]
+        ] = self._filter_expectation_configurations_by_column_type(
+            expectation_configurations, include_column_names, exclude_column_names
+        )
```

Comment:
Helper method to filter relevant column-based expectation configurations from the list - the result is a dictionary with keys representing expectation name and values representing the list of relevant configs.